lib/internal/test_runner/utils.js

```diff
@@ -65,7 +64,7 @@ function createDeferredCallback() {
 }
 
 function isTestFailureError(err) {
-  return err?.code === 'ERR_TEST_FAILURE' && kIsNodeError in err;
+  return err instanceof ERR_TEST_FAILURE;
```

Comment:
Likely

Reply:
then could it be modified to not be?

Reply:
The alternative is actually easier to confuse: just checking for a string as name is very error prone. Changing the instance would however require quite some work to do and seems less error prone. I therefore opt for keeping this check as it is.

Reply:
If node is the only creator of ERR_TEST_FAILURE errors, then it could keep a WeakSet of all the instances created, and this check could test for membership in that WeakSet. It seems like a 2-4 line change to make it zero error prone.